lib/setup/setupList.js

```diff
@@ -189,7 +189,7 @@ function List(options) {
             filter = null;
         }
         if (filter) {
-            filter = filter.replace('*', '');
+            filter = filter.replace(/\*/g, '');

```

Comment:
I assume we want to replace all occurences, not only the first?

Reply:
anyway only one is supported